code/datums/cogbar.dm

```diff
@@ -0,0 +1,87 @@
+#define COGBAR_ANIMATION_TIME 5
```

Comment:
```suggestion
#define COGBAR_ANIMATION_TIME 5 DECISECONDS
```

let's be clear

Reply:
nope! just copy pasted over from progbars